web/client/utils/QueryParamsUtils.js

```diff
@@ -83,3 +83,56 @@ export const postRequestLoadValue = (name, storage = sessionStorage) => {
 export const getRequestParameterValue = (name, state, storage = sessionStorage) => {
     return getRequestLoadValue(name, state) ?? postRequestLoadValue(name, storage);
 };
+
+
+/**
+ * Map a set of URL querystrings to a KVP object
+ * where each is single key is the parameter and the value is the parameter value
+ * mapping is based on an object that maps each query string param to a redux action
+ * @param {object} paramActions objects that maps each parameter to its respective action to trigger
+ * @param {object} state the application state
+ * @returns {object} { param: value } KVP object
+ */
+export const getParametersValues = (paramActions, state) => (
+    Object.keys(paramActions)
+        .reduce((params, parameter) => {
+            const value = getRequestParameterValue(parameter, state);
```

Comment:
I'm not sure yet but the failing tests could be related to use of sessionStorage in the getRequestParameterValue function that result different in the master env. The only way to verify that is to debug it locally and check if the sessionStorage has different content when it reach the failing unit tests comparing naster with 2022.01.xx branches. We should ensure that the unit test contains all the needed data and does not rely on configuration composed in other unit tests

Reply:
It is ok for the investigation. If the root cause is the use of `session storage` I may need the help of the dev who wrote that method in the first place (@alexander-fedorenko)

Reply:
from `master` session storage is:

```JSON 
{"queryParams":"{"featureinfo":{"lat":5,"lng":5,"filterNameList":[]},"zoom":3,"center":"41,0"}"}
```

Reply:
@allyoucanmap @tdipisa as discussed, the bug that made tests fail was due to lines of code missing from ported branch, those have been reinstated, diffing with `master` (where these lines are there) and @allyoucanmap saw the result and he was happy with it. 

As for the `export const paramActions ...` this will probably require further discussion with other devs to consider the best place where to place `paramActions` object, the epics file where it is now might not be the most appropriate file, it is probably better to move the login the `QueryParamUtils` file.